Automattic/wp-calypso
# Social Connections: Fix bug with service examples

Closes https://github.com/Automattic/jetpack-reach/issues/711
Fixes a bug introduced in https://github.com/Automattic/wp-calypso/pull/96605

## Proposed Changes

* Updates `SERVICES_WITH_EXAMPLES` to have all the services.

## Why are these changes being made?


* In the mentioned eariler change the logic on when a panel can be expanded was changed, which introduced a hiding bug; The `SERVICES_WITH_EXAMPLES` variable was not up to date with all the services. This made the panel non-expandable even that all our current services have examples now.

## Testing Instructions



* Check that every service from the file is included in the list
* Go to `/sites/marketing/connections/<SITE_URL>`, and have every connection disconnected.
* Make sure you can expand each connection and see the examples

BEFORE:
![image](https://github.com/user-attachments/assets/7ed279ea-637f-4e80-9e15-4df6d387c132)

AFTER:

https://github.com/user-attachments/assets/d00515d8-3db0-4296-90a5-821849c792f8

## Pre-merge Checklist
